README.md

```diff
@@ -1,3 +1,5 @@
+<a name="readme-top"></a>
```

Comment:
I merged back to top with this PR, so please remove it.
https://github.com/yamadashy/repopack/pull/116